venues/OpenReview.net/Support/revisionProcess.py

```diff
@@ -1,3 +1,91 @@
+
+    elif invitation_type == 'Meta_Review_Stage':
+        meta_review_due_date = None
+        if forum_note.content.get('meta_review_deadline', None):
+            try:
+                meta_review_due_date = datetime.datetime.strptime(forum_note.content.get('meta_review_deadline', None), '%Y/%m/%d %H:%M')
+            except ValueError:
+                meta_review_due_date = datetime.datetime.strptime(forum_note.content.get('meta_review_deadline', None), '%Y/%m/%d')
+
+        meta_review_start_date = None
+        if forum_note.content.get('meta_review_start_date', None):
+            try:
+                meta_review_start_date = datetime.datetime.strptime(forum_note.content.get('meta_review_start_date', None), '%Y/%m/%d %H:%M')
+            except ValueError:
+                meta_review_start_date = datetime.datetime.strptime(forum_note.content.get('meta_review_start_date', None), '%Y/%m/%d')
+
+        meta_review_additional_fields = forum_note.content.get('additional_meta_review_options', {})
+        if meta_review_additional_fields:
+            meta_review_additional_fields = json.loads(meta_review_additional_fields)
+
+        conference.set_meta_review_stage(openreview.MetaReviewStage(start_date = meta_review_start_date, due_date = meta_review_due_date, public = (forum_note.content.get('make_meta_reviews_public', None) == 'Yes'), additional_fields = meta_review_additional_fields))
+
+    elif invitation_type == 'Decision_Stage':
+        decision_due_date = None
+        if forum_note.content.get('decision_deadline', None):
+            try:
+                decision_due_date = datetime.datetime.strptime(forum_note.content.get('decision_deadline', None), '%Y/%m/%d %H:%M')
+            except ValueError:
+                decision_due_date = datetime.datetime.strptime(forum_note.content.get('decision_deadline', None), '%Y/%m/%d')
+
+        decision_start_date = None
+        if forum_note.content.get('decision_start_date', None):
+            try:
+                decision_start_date = datetime.datetime.strptime(forum_note.content.get('decision_start_date', None), '%Y/%m/%d %H:%M')
+            except ValueError:
+                decision_start_date = datetime.datetime.strptime(forum_note.content.get('decision_start_date', None), '%Y/%m/%d')
+
+        decision_options = forum_note.content.get('decision_options', None)
+        if decision_options:
+            decision_options = [s.translate(str.maketrans('', '', ' "\'')) for s in decision_options.split(',')]
```

Comment:
This is confusing to me.  Overall you're making the string into a list, but I don't get what's going on with the translate, the quotes don't match up. '"\''

Reply:
Here, I am splitting the input string decision_options at ',' and then from each output of the split, I am removing spaces, and quote chars ( " and ' ). 
The quotes do not have to match up here as the double quote is inside a single quote space and the following single quote is escaped.

Reply:
Now that I think about it, maybe removing spaces is not a good idea. I should instead use string.strip() for this to shave off the spaces from the ends.